GPU/Common/DepthBufferCommon.h

```diff
@@ -0,0 +1 @@
+#pragma once
```

Comment:
Hm, nothing imports this though?

-[Unknown]

Reply:
Oh, I was gonna move stuff here, then didn't.